### PR TITLE
chore: update react-native from 0.60.x to 0.61.x

### PR DESCRIPTION
**Pull request by patogallaiovlabs**
## Description

From previous step 0.59->0.60 (#690), this step goes from 0.60 to 0.61.
Changelog of 0.61: https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#v0610
We followed this guide:     
- https://react-native-community.github.io/upgrade-helper/?from=0.60.6&to=0.61.5
- https://mattoakes.net/react-native-upgrade-guides/upgrade-to-react-native-0.61/

Common issues solved here:
- Images not showing: https://stackoverflow.com/a/64152729/14606958
- Xcode 12.5 not working (Pod issues): https://infinitbility.com/build-failed-after-update-xcode-12.5 

## How Has This Been Tested?

Doing some basics interactions, txs send, etc.

## Types of changes

- [ ] fix: bug fixes, e.g. fix Button color on DarkTheme.
- [ ] feat: new features, e.g. add Snackbar component.
- [ ] refactor: code refactor, e.g. new folder structure for components.
- [ ] docs: changes into documentation, e.g. add usage example for Button.
- [ ] test: adding or updating tests, eg unit, snapshot testing.
- [x] chore: tooling changes, e.g. change circleci config.
- [x] BREAKING CHANGE: for changes that break existing usage, e.g. change API of a component.

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)

## NPM Audit
- **Before** this PR:
	- found 239 vulnerabilities (26 low, 154 moderate, 59 high) in 2179 scanned packages
- **After** this PR:
	- found 238 vulnerabilities (25 low, 154 moderate, 59 high) in 2197 scanned packages

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
       android:icon="@mipmap/ic_launcher"
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
+      android:usesCleartextTraffic="true"
```
Comment (martinmedina) [inline android/app/src/main/AndroidManifest.xml:22]: This new flag added and set to true caught my attention. I believe that we should only be working with HTTPS files so it should not be needed. @patogallaiovlabs what are your thoughts?

**Review by martinmedina — COMMENTED** [review]
The only comment made so far reflects what I found while checking this PR. I'll give it a final look once the comment is addressed.

**Comment by lucachaco** [comment]
tRBTC sent from android
image

**Comment by lucachaco** [comment]
tRBTC sent with IOS


image

**Review by lucachaco — APPROVED** [review]
LGTM. Works on ios and android

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (martinmedina) [reply android/app/src/main/AndroidManifest.xml]: Flag removed in #707 which will be part of the same release (2.0.0) as this PR. All good!

**Review by martinmedina — APPROVED** [review]
LGTM!

**Pull request merged**